docker/docker-compose.yml

```diff
@@ -91,6 +91,8 @@ services:
   pinot:
     image: hypertrace/pinot-servicemanager:main
     container_name: pinot
+    ports:
+      - 9000:9000
```

Comment:
this will re-introduce a port that commonly conflicts, literally why we had the debate about port 2020 if you remember. could it be a better choice to document how to map an exposed port?